android-components/components/browser/menu2/src/main/res/values/dimens.xml

```diff
@@ -4,7 +4,7 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <resources>
     <dimen name="mozac_browser_menu2_corner_radius">4dp</dimen>
-    <dimen name="mozac_browser_menu2_elevation">8dp</dimen>
+    <dimen name="mozac_browser_menu2_elevation">5dp</dimen>
```

Comment:
[Link](https://www.figma.com/file/epu0Sj6mCPk7K282sNpHgI/Site?node-id=3850%3A59494&t=M9Wqnn9vei1aRxTI-0) to the design. Having less then 8 came in handy, because getting the [4dp offsets](https://www.figma.com/file/epu0Sj6mCPk7K282sNpHgI/Site?node-id=3850%3A60276&t=M9Wqnn9vei1aRxTI-4) with down orientation is impossible – 8 elevation means also 8dp spacing, and there is just 6dp towards the top edge of the rootview.